fili-core/src/test/java/com/yahoo/bard/webservice/models/druid/client/impl/TestDruidWebService.java

```diff
@@ -86,10 +89,11 @@ public TestDruidWebService() {
     /**
      * If TestDruidWebService#throwable is set, invokes the failure callback.  Otherwise invokes success or failure
      * dependent on whether TestDruidWebService#statusCode equals 200.
+     * Note that since this doesn't send requests to druid all the responses will be null or {@link FailedFuture}.
      */
     @Override
     @SuppressWarnings("checkstyle:cyclomaticcomplexity")
-    public void postDruidQuery(
+    public Future<org.asynchttpclient.Response> postDruidQuery(
```

Comment:
Actually, it's because this class already has a `Response` object imported (ie. `javax.ws.rs.core.Response`). Because of that, one of them has to be fully qualified